pkg/kubelet/dockershim/libdocker/fake_client.go

```diff
@@ -798,7 +798,7 @@ func createImageFromImageInspect(inspect dockertypes.ImageInspect) *dockertypes.
 
 // dockerTimestampToString converts the timestamp to string
 func dockerTimestampToString(t time.Time) string {
-	return t.Format(time.RFC3339Nano)
+	return t.UTC().Format(time.RFC3339Nano)
```

Comment:
how is this method used? what are the implications of changing from local to utc here?

Reply:
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/kubelet/dockershim/libdocker/fake_client.go#L527

It is only used for fakeclient time generation, it is optional, if we expect the time zone to be UTC, then it should be possible to add it

Reply:
if this isn't related to persisting data in the API, drop it from this PR